_data/language.yml

```diff
@@ -23,5 +23,5 @@ edit: "Edit"
 this_content_is_open_source: "This content is open source."
 help_improve_it: "Help improve it"
 enter_search_term: "Enter search term and hit enter"
-upcoming_workshops: "Upcoming Carpentries Workshops"
-past_workshops: "Past Carpentries Workshops"
+upcoming_workshops: "Upcoming Workshops Related to HPC Carpentry"
```

Comment:
Why not "Upcoming HPC Carpentry Workshops" and "Past HPC Carpentry Workshops"?

Reply:
This is a deliberate choice, an "HPC Carpentry workshop" has no clear definition and I would definitely prefer to have more relevant workshops listed here than less.